tests/test_parsers/test_grobid_full_text_parser.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import os
+import pathlib
+import unittest
+import unittest.mock as um
+
+import pytest
+from mmda.parsers.grobid_full_text_parser import GrobidFullTextParser
+from mmda.parsers.pdfplumber_parser import PDFPlumberParser
+
+os.chdir(pathlib.Path(__file__).parent)
+
+PDF_PATH = "../fixtures/grobid_full_text_parser/0c027af0ee9c1901c57f6579d903aedee7f4.pdf"
+XML_OK = open("../fixtures/grobid_full_text_parser/0c027af0ee9c1901c57f6579d903aedee7f4.xml").read()
+TEST_CONFIG_PATH =  "../fixtures/grobid_full_text_parser/grobid.config"
+
+def mock_request(*args, **kwargs):
+    class MockResponse:
+        def __init__(self, xml: str, status_code: int) -> None:
+            self._xml = xml
+            self._status_code = status_code
+
+        @property
+        def text(self):
+            return self._xml
+
+        @property
+        def status_code(self):
+            return self._status_code
+
+    return MockResponse(XML_OK, 200)
+
+
+class TestGrobidFullTextParser(unittest.TestCase):
+    @um.patch("requests.request", side_effect=mock_request)
+    def test_processes_full_text(self, mock_request):
+        logging.getLogger("pdfminer").setLevel(logging.WARNING)
```

Comment:
wats goin on here?

Reply:
I'm not sure why, but when I ran pytest on this file, a bunch of pdfminer DEBUG logs were popping up, so google told me to do this to turn it down a notch.

Reply:
ah gotcha! i think i kno wats happening here, thx for the solution, might actually add this closer to where pdfminer appears